docfx_yaml/extension.py

```diff
@@ -959,10 +961,19 @@ def process_docstring(app, _type, name, obj, options, lines):
     app.env.docfx_info_uid_types[datam['uid']] = _type
 
 
+# Uses black.format_str() to reformat code as if running black/linter
+# for better presnetation.
+def format_code(code):
+    # Signature code comes in raw text without formatting, to run black it
+    # requires the code to look like actual function declaration in code.
+    # Returns the original formatted code without the added bits.
+    return black.format_str("def " + code + ": pass", mode=black.FileMode())[4:-11]
```

Comment:
Nice!

Reply:
I like this too, but it might be fragile. How is `code` generated?

The problem seems to be with representations of default values that are not `eval()`-uable, such as the default timeout object.

An [error](https://source.cloud.google.com/results/invocations/38ad16c2-d98a-49b4-88a7-c90163dcf7b1/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fdocs%2Fdocs-presubmit/log) from a bigquery build:
```
Cannot parse: 1:58: def cancel_job(job_id, project=None, location=None, retry=<google.api_core.retry.Retry object>, timeout=None): pass
```

Reply:
The `code` is found from `signature` variable at this point: https://github.com/googleapis/sphinx-docfx-yaml/blob/4e1d03fe1bc78a67656a3f71fb93981e8fd3e255/docfx_yaml/extension.py#L984

Which is passed from Sphinx in https://github.com/sphinx-doc/sphinx/blob/4.x/sphinx/ext/autodoc/__init__.py#L498. At this point it seems like we can't resolve the references so I've filed #151 to look into this.